OpenRobertaServer/src/test/resources/crossCompilerTests/_expected/robotSpecific/targetLanguage/nxt/text_colours_messages_functions.nxc

```diff
@@ -142,10 +142,10 @@ void messages() {
     NumOut(___numberVar, (MAXLINES - ___numberVar) * MAXLINES, BluetoothGetNumber(9));
     BoolOut(___numberVar, (MAXLINES - ___numberVar) * MAXLINES, BluetoothGetBoolean(9));
     TextOut(___numberVar, (MAXLINES - ___numberVar) * MAXLINES, BluetoothGetString(9));
-    BoolOut(___numberVar, (MAXLINES - ___numberVar) * MAXLINES, (BluetoothStatus(0)==NO_ERR));
-    BoolOut(___numberVar, (MAXLINES - ___numberVar) * MAXLINES, (BluetoothStatus(1)==NO_ERR));
-    BoolOut(___numberVar, (MAXLINES - ___numberVar) * MAXLINES, (BluetoothStatus(2)==NO_ERR));
-    BoolOut(___numberVar, (MAXLINES - ___numberVar) * MAXLINES, (BluetoothStatus(3)==NO_ERR));
+    TextOut(___numberVar, (MAXLINES - ___numberVar) * MAXLINES, (BluetoothStatus(0)==NO_ERR));
+    TextOut(___numberVar, (MAXLINES - ___numberVar) * MAXLINES, (BluetoothStatus(1)==NO_ERR));
+    TextOut(___numberVar, (MAXLINES - ___numberVar) * MAXLINES, (BluetoothStatus(2)==NO_ERR));
+    TextOut(___numberVar, (MAXLINES - ___numberVar) * MAXLINES, (BluetoothStatus(3)==NO_ERR));
```

Comment:
I found the reason for this:
Please have a look at `NxtNxcVisitor#L388`, there you will see a check with the `toString` method for `CapturedType` expressions, with a check if the toString contains `BluetoothConnectAction`. Please change this.